server/sonar-server/src/main/java/org/sonar/server/issue/IssueQueryService.java

```diff
@@ -251,7 +252,7 @@ private Date buildCreatedAfterFromRequest(DbSession dbSession, SearchWsRequest r
     String uuid = allComponentUuids.iterator().next();
     // TODO use ComponentFinder instead
     ComponentDto component = checkFoundWithOptional(componentService.getByUuid(uuid), "Component with id '%s' not found", uuid);
-    SnapshotDto snapshot = dbClient.snapshotDao().selectLastSnapshotByComponentId(dbSession, component.getId());
+    SnapshotDto snapshot = dbClient.snapshotDao().selectLastSnapshotByComponentId(dbSession, firstNonNull(component.parentProjectId(), component.getId()));
```

Comment:
component.parentProjectId is returning the first module after the project, is it really what you want ?
Because it will work for simple project, but for multi module project I suspect that it will not returned what you want.

In fact this field should not be used, as it error prove (and there's a ticket to remove it).
You should do that with 2 queries : one to get the snapshot from the component, then get the root snapshot from the snaphot.